### PR TITLE
Add production ready Dockerfile

### PR DESCRIPTION
**Pull request by zirkome**
Hi FreshRSS community,

Two days ago I stumbled upon the FreshRSS' `Dockerfile` for development purpose but I needed it for production purpose. Facing the lack of FreshRSS images I made it myself. At first I made a separate repository but I think the `Dockerfile` belongs to the project itself and should be directly available in the source repository.

This PR add a `Dockerfile` that build a PHP/Apache image that runs FreshRSS and the update of the README to help running or build the image at home.

Thanks :)

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks for that :+1: 
I will let @marienfressinaud have a look. It can either be in the FreshRSS repository, or in https://github.com/FreshRSS/docker-freshrss (e.g. different branches or different files for master, beta, dev). https://github.com/FreshRSS/docker-freshrss/issues/3

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/issues/520

**Comment by marienfressinaud** [comment]
I will have a look during the next few days, thanks to your contribution :)

**Comment by marienfressinaud** [comment]
Sorry for the delay: I will create a new repository for the Dockerfile and push it on https://hub.docker.com/ (maybe with a "FreshRSS" account instead of mine).

Could you make your PR for documentation on the corresponding repo (i.e. https://github.com/FreshRSS/documentation, in `./{fr,en}/docs/users/01_Installation.md`, after the Nginx section)? I would like to keep the README quite short.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM php:5.6-apache
+MAINTAINER Guillaume Fillon <guillaume@posteo.de>
+
+
+RUN apt-get update && \
+    apt-get install -y libcurl4-openssl-dev libjpeg62-turbo-dev \
+    libmcrypt-dev libpng12-dev libicu-dev libgmp-dev libsqlite3-dev && \
+    ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h && \
```
Comment (Alkarex) [inline Dockerfile:8]: I have not tried myself yet, but isn't it possible to have a command a bit more standard, such as 

```
apt-get install php5-curl php5-gd php5-intl php5-json php5-gmp php5-sqlite
```

A bit like in the readme https://github.com/FreshRSS/FreshRSS/blob/beta/README.md#example-of-full-installation-on-linux-debianubuntu

**Comment by Alkarex** [comment]
See also https://github.com/linuxserver/docker-freshrss

**Comment by bnjbvr** [comment]
Hey, I'd like to revive this PR: I've made a Dockerfile based upon yours, and it seems to work great at the moment. Is there a way we could move this forward? Either take the PR as is and I'll base my PR over this one, or if @kokaz is OK, I could take it over?

**Comment by Alkarex** [comment]
Thanks @bnjbvr 
I did not have time yet to look at it myself, but it would be nice to finalize a Docker file indeed.

**Comment by marienfressinaud** [comment]
btw, it would be cool to have it in https://github.com/FreshRSS/docker-freshrss-production ;)

**Comment by zirkome** [comment]
Hey @bnjbvr,
Yeah of course you can take this PR to the next level :) I don't have much time to deal with it anyway so whatever you want with it!

**Comment by bnjbvr** [comment]
Cheers! I'll try to make a PR to the other repository, then.

**Comment by zirkome** [comment]
Do I close the PR?

**Comment by bnjbvr** [comment]
Yeah. I'll give credit to your work in the new Dockerfile, if that fits you (as my docker image is based upon yours).

**Comment by zirkome** [comment]
@bnjbvr I'm OK with that :) Thanks for your contribution

**Pull request closed without merging**